controller/contour/controller.go

```diff
@@ -286,3 +286,28 @@ func contourFinalized(contour *operatorv1alpha1.Contour) bool {
 
 	return false
 }
+
+// doOwnerLabelsExist returns true if obj contains Contour owner labels.
+func doOwnerLabelsExist(obj runtime.Object, contour *operatorv1alpha1.Contour) bool {
+	labels := obj.(metav1.Object).GetLabels()
```

Comment:
can this function take a `metav1.Object` directly instead of a `runtime.Object`, so we don't have to type-assert here?